experiments/jacobian-lens-0.5b/compare_columns.py

```
@@ -0,0 +1,80 @@
+                          "only_a": len(set(a) - set(b)),
+                          "only_b": len(set(b) - set(a))}))
+        return 1
+    positions = 0
+    values = 0
+    exact = 0
+    max_abs = 0.0
+    worst = None
+    for key in sorted(a):
+        va, vb = a[key], b[key]
+        positions += 1
+        for layer, (la, lb) in enumerate(zip(va, vb)):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)